src/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
>We could compile this xunit directive into only the netfx test build then, if it makes a measurable impact on test run time.

We don't make NETFX specific builds of things anymore. We tend to build all tests against 'netstandard' (which is how our customers will build things). So, we can't do conditional compilation like that.  We could perhaps change that but then we wouldn't be testing against 'netstandard' on top of NETFX....we would be testing a different configuration build against NETFX.